Runtime/ModularAvatarGlobalCollider.cs

```diff
@@ -0,0 +1,58 @@
+﻿#region
+
+using UnityEngine;
+
+#endregion
+
+namespace nadena.dev.modular_avatar.core
+{
+	//All available VRChat Colliders
+	public enum GlobalCollider
+	{
+		Head,
+		Torso,
+		HandLeft, HandRight,
+		FingerIndexLeft, FingerIndexRight,
+		FingerMiddleLeft, FingerMiddleRight,
+		FingerRingLeft, FingerRingRight,
+		FingerLittleLeft, FingerLittleRight,
+		FootLeft, FootRight,
+		None,
+	}
+
+	[AddComponentMenu("Modular Avatar/MA Global Collider")]
+	[HelpURL("https://modular-avatar.nadena.dev/docs/reference/global-collider?lang=auto")]
+	public class ModularAvatarGlobalCollider : AvatarTagComponent
+	{
+		//Toggle for manual remapping, if false auto remapping.
+		public bool manualRemap = false;
+
+		//descriptor collider to replace (Only used in manual mode)
+		public GlobalCollider sourceCollider;
+
+		//May be better to do this with the method bone proxy uses
+		public AvatarObjectReference rootTransform = new AvatarObjectReference();
+		public GameObject remapTargetObject => rootTransform.Get(this);
```

Comment:
This is fine, if the goal is to be analogous with physbone references. I would suggest something more like this though:

```
internal AvatarObjectReference m_rootTransform = new();
public Transform RootTransform {
  get => m_rootTransform?.Get(this)?.transform ?? transform,
  set {
    if (m_rootTransform == null) m_rootTransform = new();
    m_rootTransform.set(value?.gameObject);
  }
}
```